python/tskit/tables.py

```diff
@@ -2493,6 +2493,7 @@ def simplify(
         filter_individuals=True,
         filter_sites=True,
         keep_unary=False,
+        keep_unary_in_individuals=False,
```

Comment:
Let's make the default in the signature ``None``, will give us some room to change the API later.

Reply:
Do you want the default for `keep_unary` to be `None` too? This would be a change from the current API, but shouldn't actually change any behaviour.